hugo/content/downloads.md

```diff
@@ -73,6 +73,8 @@ document.getElementById('suggested-download').innerHTML = getPlatformContent(get
 {{< content-layout/download name="Static Linux Server" href="linux-static-server" osclass="linux">}}
 {{< /content-layout/downloads >}}
 
+**Note:** The Static Linux Server is using an outdated version of OpenSSL - see [#4001](https://github.com/mumble-voip/mumble/issues/4001) for details. 
```

Comment:
```suggestion
**Note:** The static Linux server is using an outdated version of OpenSSL - see [#4001](https://github.com/mumble-voip/mumble/issues/4001) for details. 
```

Reply:
Well, the `Static Linux Server` is the exact name of the package you provide, so I think it can and maybe even should be capitalized. I could add a code tag like `Static Linux Server`.

Reply:
In that case I'd go without the leading "The" though:
```suggestion
**Note:** `Static Linux Server` is using an outdated version of OpenSSL - see [#4001](https://github.com/mumble-voip/mumble/issues/4001) for details. 
```

Reply:
It's not perfect, but I guess we could use that.